src/components/block/FooterCTACard/FooterCTACard.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import FooterCTACard from "./FooterCTACard";
+
+export default {
+  title: "Block/FooterCTACard",
+  component: FooterCTACard,
+};
+
+const Template = ({ subtitle, body, buttonText }) => (
+  <div style={{ backgroundColor: "black", width: "480px" }}>
+    <FooterCTACard subTitle={subtitle} body={body} buttonText={buttonText} />
+  </div>
+);
+export const DefaultFooterCTACard = Template.bind({});
+DefaultFooterCTACard.args = {
+  subtitle: "Get Involved",
+  body: "Volunteer with us and put your professional skills to good use.",
+  buttonText: "JOIN US",
```

Comment:
This text should not be in all caps because the button component itself is the one that is setting the [`text-transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform) property to always uppercase the text. In other words, the all-caps-ness is part of the style, not the content.